cmd/dev/embed_toml.cpp

```diff
@@ -82,11 +97,9 @@ int main(int argc, char* argv[]) {
             output << "#include <array>\n\n";
             output << "#include <silkworm/node/snapshots/entry.hpp>\n\n";
             output << "namespace silkworm::snapshots {\n\n";
-            output << "inline constexpr std::array<Entry, " << table.size() << "> k" << snapshot_name << "Snapshots{\n";
-            for (auto&& [key, value] : table) {
-                std::string key_str{key.begin(), key.end()};
-                std::string val_str{value.as_string()->get()};
-                output << "    Entry{\"" << key_str << "\"sv, \"" << val_str << "\"sv},\n";
+            output << "inline constexpr std::array<Entry, " << filtered_table.size() << "> k" << snapshot_name << "Snapshots{\n";
```

Comment:
Can be simply `std::array` without the template arguments (they are auto-deducted).